src/ice_thermo_oce.F90

```diff
@@ -10,11 +10,33 @@ subroutine cut_off()
 
 where(a_ice<0.1e-8_WP)
  a_ice=0.0_WP
+ m_ice=0.0_WP
+ m_snow=0.0_WP
+#if defined (__oifs)
+ ice_temp=273.15_WP
+#endif /* (__oifs) */
 end where
 
 where(m_ice<0.1e-8_WP)
  m_ice=0.0_WP
+ m_snow=0.0_WP
+ a_ice=0.0_WP 
```

Comment:
will destroy reproducibility!

Reply:
Ty, I'll just pull it into the ifdef oifs